tests/countries/test_united_states_virgin_islands.py

```
@@ -55,3 +47,66 @@ def test_government_holidays(self):
+            ("2022-07-04", "Independence Day"),
+            ("2022-09-05", "Labor Day"),
+            ("2022-10-10", "Columbus Day; Columbus Day and Puerto Rico Friendship Day"),
+            ("2022-10-31", "Halloween"),
+            ("2022-11-01", "Liberty Day"),
+            ("2022-11-11", "Veterans Day"),
+            ("2022-11-24", "Thanksgiving Day"),
+            ("2022-12-25", "Christmas Day"),
+            ("2022-12-26", "Christmas Day (observed); Christmas Second Day"),
+        )
+
+    def test_l10n_th(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)